### PR TITLE
Some links to redirect people and RaspberryPi Install!

### PR DESCRIPTION
**Pull request by kylerschin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
...
+
+Your gonna need these packages: `
+apt-get install nodejs build-essential git`
+
+Next, grab the repository and build.
+
+    cd /opt
+    git clone https://github.com/cjdelisle/cjdns.git
+    cd cjdns
+    NO_TEST=1 Seccomp_NO=1 ./do`
```
Comment (benhylau) [inline README.md]: `NO_TEST=1` isn't needed anymore in v18

Comment (benhylau) [inline README.md]: The CFLAGS are necessary for some 10+ times performance gain if I remembered correctly: https://github.com/tomeshnet/prototype-cjdns-pi2/blob/master/scripts/install#L8-L28

File: README.md [hunk 2]
```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
 #### Debian-based distro:
 
     sudo apt-get install nodejs git build-essential python2.7
+    #Raspberry Pi Cjdns Install
```
Comment (benhylau) [inline README.md]: Format

File: README.md [hunk 3]
```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
...
+
+##### Raspberry Pi
```
Comment (benhylau) [inline README.md]: I don't think sticking a huge section for Pi right in the middle of the README.md is a good idea. I suggest modifying https://github.com/tomeshnet/prototype-cjdns-pi2/blob/master/scripts/install into a scripted install and stick it in **contrib**, then add a line in the README to just run that

File: README.md [hunk 4]
```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
...
+    cd cjdns
+    NO_TEST=1 Seccomp_NO=1 ./do`
+
+The last line is due to a issue with SecComp on Raspbian Kernal seen here:
+https://github.com/hyperboria/bugs/issues/6#issuecomment-162244016
+
+After a little wait, the build should finish successfully. Now we want to configure cjdns to run as a daemon, so let’s create a link to the binary, generate a configuration file, and copy over the service file.
+
+    ln -s /opt/cjdns/cjdroute /usr/bin
+    (umask 077 && ./cjdroute --genconf > /etc/cjdroute.conf)
```
Comment (benhylau) [inline README.md]: This is done by **cjdns.service** on first run

File: README.md [hunk 5]
```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
...
+    cp contrib/systemd/cjdns.service /etc/systemd/system/
```
Comment (benhylau) [inline README.md]: You need the resume service file

**Comment by kylerschin** [comment]
Thank you so much! Will do ASAP!

**Comment by benhylau** [comment]
Thanks for adding https://n-o-d-e.net/hyperboria.html by the way, that was a nice intro @famicoman wrote.

**Comment by kylerschin** [comment]
@benhylau Should I create another README in contrib and link to that? Seems too much to me. Thanks.

File: README.md (hunk 3 above)
Comment (kylerschin) [inline README.md]: Seems fixed. I linked to the repository instead of the script. Give people for information.

Comment (benhylau) [inline README.md]: The script in the Toronto repo is quite specific and does not only install cjdns. I suggest extracting part of that script, put them in contrib here, and reference that instead.

**Comment by cjdelisle** [comment]
This looks like it might be incomplete at the moment, I'm going to close it for now but do feel free to re-open or get in touch with us if you need any help.

**Comment by kylerschin** [comment]
Sorry, Left it hanging here for a long time.

**Pull request closed without merging**